contracts/pool/src/methods/account_position.rs

```diff
@@ -27,17 +46,21 @@ pub fn account_position(env: &Env, who: &Address) -> Result<AccountPosition, Err
 pub fn calc_account_data(
     env: &Env,
     who: &Address,
-    mb_who_collat: Option<&AssetBalance>,
-    mb_who_debt: Option<&AssetBalance>,
-    mb_s_token_supply: Option<&AssetBalance>,
-    mb_debt_token_supply: Option<&AssetBalance>,
+    cache: CalcAccountDataCache,
```

Comment:
Let's also pass the `cache` as a reference, like we do for other structs across the codebase.